docs/api.rst

```diff
@@ -0,0 +1,7 @@
+API Reference
+=============
+
+.. toctree::
+   :maxdepth: 2
+
+   api/nameko
```

Comment:
not sure we want to blanket include everything here. is there much benefit over e.g. just browsing the code?